lib/perl/Genome/ModelGroup/Command/Create.pm

```diff
@@ -55,7 +55,7 @@ sub execute {
         return;
     }
     $self->status_message('Created model group:');
-    $self->status_message('ID: ' . $model_group->id . ', NAME: ' . $model_group->name);
+    $self->status_message('ID: ' . $model_group->id . '  NAME: ' . $model_group->name);
```

Comment:
It would be better if format print is used:
$self->status_message("ID: %s    NAME: %s", $model_group->id, $model_group->name);